packages/host/app/components/operator-mode/container.gts

```diff
@@ -625,6 +711,65 @@ export default class OperatorModeContainer extends Component<Signature> {
   </template>
 }
 
+export function getCardStackItem(
+  stackItem: StackItem,
+  stackItems: StackItem[]
+): CardStackItem {
+  if (stackItem.type === 'card') {
+    return stackItem;
+  }
+  if (stackItem.fieldOfIndex >= stackItems.length) {
+    throw new Error(
+      `bug: the stack item (index ${stackItem.fieldOfIndex}) that is the parent of the contained field '${stackItem.fieldName}' no longer exists in the stack`
+    );
+  }
+  return getCardStackItem(stackItems[stackItem.fieldOfIndex], stackItems);
+}
```

Comment:
Is this meant to recurse to an arbitrary depth?? Im not sure its recursing correctly. It can only look back to one level deep from what I see because the 2nd argument is not changing

Reply:
Absolutely, because contains fields can be nested (there is an integration test for this too that shows this works). The 2nd argument changed in line 726

Reply:
(We can rename `stackItems` to `stack` if that helps comprehension)

Reply:
ahh ok my bad I understand this now. Somehow I was thinking like contained cards worked across stacks.

- stackItems is all cards within the same stack